roles/content_view_publish/tasks/main.yml

```diff
@@ -13,6 +13,8 @@
     force_promote: "{{ content_view.force_promote | default(omit) }}"
     force_yum_metadata_regeneration: "{{ content_view.force_yum_metadata_regeneration | default(omit) }}"
     current_lifecycle_environment: "{{ content_view.current_lifecycle_environment | default(omit) }}"
+    async: "{{ content_view_async | default(omit) }}"
+    poll: "{{ content_view_poll | default(omit) }}"
```

Comment:
These variables should be called `foreman_content_view_publish_async` and `foreman_content_view_publish_poll`